src/scss/components/_previous-link.scss

```diff
@@ -0,0 +1,17 @@
+@use 'microscope-sass/lib/bem';
+
+.openforms-previous-link {
+  @include bem.modifier('start') {
+    margin-block-end: 20px;
```

Comment:
hrm, block-level elements should never define margins, since this breaks how you use a component inside other components (it should be limited to paddings). Can we do a different approach based on flexbox/grid where the row-gaps are used for spacing between blocks?

Reply:
Mhh I think this would introduce a lot of changes for the form elements, but i'll take a look (that is precisely why is added the margin 😄 )

Reply:
Lets discuss this tomorrow, this will limit some customization (for example the `--of-summary-progress-padding-block-end` design token will be obsolete. And if we would make the `row-gap` on the form step elements customizable, it would change the spacing between all elements)

Reply:
let's be pragmatic and go with the margin then for now, but please add a comment like:

```js
// NOTE: usually block elements should not define margins, but this is done deliberately
// See https://github.com/open-formulieren/open-forms-sdk/pull/718#discussion_r1791617060 for
// more discussion context.
```